STMicroelectronics/STM32L4_L4+/app/az_ulib_dm/src/az_ulib_dm.c

```diff
@@ -69,7 +70,29 @@ AZ_NODISCARD az_result az_ulib_dm_init(az_ulib_dm* dm_handle)
     az_span_fill(package->name, '\0');
   }
 
-  return _az_ulib_dm_interface_publish();
+  az_result result;
+
+  // Initialize device registry
+  result = az_ulib_registry_init();
+
+  // test add to registry
+  az_span key = az_span_create_from_str("samplekey1samplekey1samplekey1samplekey1samplekey1samplekey1samplekey1samplekey1samplekey1samplekey1samplekey1samplekey1samplekey1samplekey1samp");
+  az_span value = az_span_create_from_str("samplevalue1samplevalue1samplevalue1samplevalue1samplevalue1samplevalue1samplevalue1samplevalue1samplevalue1samplevalue1samplevalue1samplevalue1samplevalue1");
```

Comment:
Are these large enough to be pushing max lengths? If so can you add a comment noting what that length is?

Reply:
Theoretically the longest one string can be is the entire space of the flash space allocated for the registry(4k) but Marcos has recommended an out of memory warning that should solve this issue.